docs/src/extend/firmware-management/building-image/yocto/tutorial.md

```diff
@@ -17,6 +17,10 @@ Whilst the methodology described on this page can be extended to support other d
 
 ## Setting up your build system
 
+:::info
+The instructions below were tested with the `kirkstone` release. If you want to use a different project version or check the most recent requirements, visit [Official Yocto quick guide](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html). Make sure you select the proper release, as the instructions may vary for different versions.
```

Comment:
I don't think this statement goes far enough to highly the importance of matching the yocto version that is supported by the meta-tedge-projects. The rewording won't be trivial, but I'll give it a shot soon.